### PR TITLE
Index style

### PR DESCRIPTION
**Pull request by MichaelvMorrison**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -15,7 +15,7 @@
 </head>
 
 <body>
-  <?php include 'nav_bar.php';?>
+  <?php include 'nav_bar.php'?>
```
Comment (lramos15) [inline index.php]: Why did you remove the semicolon?

Comment (MichaelvMorrison) [reply index.php]: felt like it :0
